carbon/assetlist.json

```diff
@@ -61,67 +52,33 @@
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/carbon/images/usc.png",
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/carbon/images/usc.svg"
       },
-      "images": [
-        {
-          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/carbon/images/usc.png",
-          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/carbon/images/usc.svg",
-          "theme": {
-            "primary_color_hex": "#053e4f"
-          }
-        }
-      ]
+      "coingecko_id": "usd-coin"
     },
     {
-      "description": "BNB token on Carbon",
+      "description": "Grouped USD on Carbon",
       "denom_units": [
         {
-          "denom": "bnb.1.6.773edb",
+          "denom": "usd",
```

Comment:
you cannot have multiple units defined as the same unit. This (0 exponent) unit must be equal to the `base` unit (cgt/1).